src/components/Profile/components/Identification/Identification.module.scss

```diff
@@ -155,7 +155,7 @@ $image-side-width-desktop: 90px;
               text-decoration: none;
             }
             &_container {
-              color: $primary-blue;
+              color: var(--mui-palette-primary-300);
```

Comment:
Is it possible to just use `--mui-palette-primary` here instead of specifying which shade of the primary color we want to use?